packages/amplify-migration-codegen-e2e/src/assertions.ts

```diff
@@ -5,50 +5,108 @@ import {
   getFunction,
   getAppSyncApi,
   describeCloudFormationStack,
+  getUserPoolClients,
 } from '@aws-amplify/amplify-e2e-core';
 import { getProjectOutputs } from './projectOutputs';
 import { getAppSyncDataSource, getResourceDetails } from './sdk_calls';
 import { removeProperties } from '.';
+import { $TSAny } from '@aws-amplify/amplify-cli-core';
 
-export async function assertGen1Setup(projRoot: string) {
+async function getMeta(projRoot: string) {
```

Comment:
seems like you don't need this wrapper function since extra logic are removed, invoke getProjectMeta() directly.

Reply:
Addressed [dbf44b4](https://github.com/aws-amplify/amplify-cli/pull/13978/commits/dbf44b417f56692be3608d684dc8f31f985b6df8)